packages/landingpage-new/styles/globals.css

```diff
@@ -32,3 +37,35 @@ ino-chip * {
   margin-bottom: 200px;
   row-gap: 80px;
 }
+
+.section-container section:first-child {
+  margin: 80px 0 0 0;
+}
+
+.inovex-elements-typo h1 {
+  font-size: 40px;
+}
+
+.inovex-elements-typo h2 {
+  font-size: 28px;
+  margin-top: 80px;
+}
+
+.table-shadow {
+  box-shadow: 0 13px 27px -5px rgba(50, 50, 93, 0.25),
+    0 8px 16px -8px rgba(0, 0, 0, 0.3), 0 -6px 16px -6px rgba(0, 0, 0, 0.025);
+  margin-bottom: 15px;
+}
+
+.content-menu-layout {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  grid-template-columns: minmax(400px, 960px) 1fr;
+}
+
+.hint {
+  color: #908da2;
+  font-size: 12px;
+  margin-top: 8px;
+  margin-bottom: 24px;
+}
```

Comment:
These styles should not be part of the globals as they are kinda specific to the guides section